app/views/programming_groups/index.html.slim

```diff
@@ -0,0 +1,23 @@
+h1 = ProgrammingGroup.model_name.human(count: 2)
+
+= render(layout: 'shared/form_filters') do |f|
+  .col-auto
+    = f.label(:exercise_id, t('activerecord.attributes.programming_group.exercise_title'), class: 'visually-hidden form-label')
+    = f.collection_select(:exercise_id, Exercise.with_programming_groups, :id, :title, class: 'form-control', prompt: t('activerecord.attributes.programming_group.exercise_title'))
+
+.table-responsive
+  table.table.mt-4
+    thead
+      tr
+        th = t('activerecord.attributes.programming_group.programming_group_id')
```

Comment:
You can make this (and potentially other columns) sortable in the backend by adding:

```rb
th = sort_link(@search, :id, t('activerecord.attributes.programming_group.programming_group_id'))
```

This might be useful to always have the newest entry first in the list.

Reply:
This is only working for the exercise_id. Maybe do I need to add for each column a filter so that it is sortable?

Reply:
With your comments below it is working now. 🙂